tests/ManagerTest.php

```diff
@@ -4,1011 +4,12 @@
-
-        $permission = $this->itemsStorage
-            ->getPermission('updatePost')
-            ->withName('createPost');
-
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage(
-            'Unable to change the role or the permission name. ' .
-            'The name "createPost" is already used by another role or permission.'
-        );
-        $manager->updatePermission('updatePost', $permission);
-    }
-
-    public function testUpdateRoleNameAlreadyUsed(): void
-    {
-        $manager = $this->createManager();
-
-        $role = $this->itemsStorage
-            ->getRole('reader')
-            ->withName('author');
-
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage(
-            'Unable to change the role or the permission name. ' .
-            'The name "author" is already used by another role or permission.'
-        );
-        $manager->updateRole('reader', $role);
-    }
-
-    public function testSeveralDefaultRoles(): void
-    {
-        $manager = new Manager(
-            new FakeItemsStorage(),
-            new FakeAssignmentsStorage(),
-            new SimpleRuleFactory(),
-        );
-        $manager
-            ->addRole(new Role('a'))
-            ->addRole(new Role('b'))
-            ->addRole(new Role('c'))
-            ->setDefaultRoleNames(['a', 'b']);
-
-        $roles = $manager->getDefaultRoles();
-
-        $this->assertCount(2, $roles);
-        $this->assertSame(['a', 'b'], array_keys($roles));
-        $this->assertSame('a', $roles['a']->getName());
-        $this->assertSame('b', $roles['b']->getName());
-    }
-
-    public function testDefaultRoleNames(): void
-    {
-        $manager = new Manager(
-            new FakeItemsStorage(),
-            new FakeAssignmentsStorage(),
-            new SimpleRuleFactory(),
-        );
-
-        $returnedManager = $manager->setDefaultRoleNames(['a', 'b']);
-
-        $this->assertSame(['a', 'b'], $manager->getDefaultRoleNames());
-        $this->assertSame($manager, $returnedManager);
-    }
-
-    public function testDefaultRolesSetWithClosure(): void
-    {
-        $manager = $this->createManager();
-
-        $manager->setDefaultRoleNames(
-            static function () {
-                return ['newDefaultRole'];
-            }
-        );
-
-        $this->assertEquals(['newDefaultRole'], $manager->getDefaultRoleNames());
-    }
-
-    public function testDefaultRolesWithClosureReturningNonArrayValue(): void
-    {
-        $manager = $this->createManager();
-
-        $this->expectException(RuntimeException::class);
-        $this->expectExceptionMessage('Default role names closure must return an array');
-
-        $manager->setDefaultRoleNames(
-            static function () {
-                return 'test';
-            }
-        );
-    }
-
-    public function testDefaultRolesWithNonArrayValue(): void
```

Comment:
Why remove this test?

Reply:
Because after adding type hinting the type checking code is no longer needed.

https://github.com/yiisoft/rbac/blob/ce83aa8038d1897f3dc71669e8999fd765588383/src/Manager.php#L499

https://github.com/yiisoft/rbac/blob/ce83aa8038d1897f3dc71669e8999fd765588383/src/Manager.php#L509

Reply:
Please, revert test, but replace string value to closure that returns non-array value (for example. `fn() => 'test'`).

Reply:
There is different test for this, it was not removed:

https://github.com/yiisoft/rbac/blob/9cec38628861432a46bd099739b87c2edd2773dc/tests/Common/ManagerTestLogicTrait.php#L814-L826

Reply:
Oh, yes. I'm missed it.